js/bitfinex.js

```diff
@@ -398,38 +398,119 @@ module.exports = class bitfinex extends Exchange {
         };
     }
 
-    async fetchTradingFees (params = {}) {
+    async fetchTradingFee (symbol, params = {}) {
         await this.loadMarkets ();
+        const safeSymbol = this.safeSymbol (symbol);
```

Comment:
We should not use `this.safeSymbol ()` here, instead we use `this.market (symbol)`.